relayer/src/chain/cosmos/version.rs

```diff
@@ -142,6 +155,37 @@ fn parse_ibc_go_version(version_info: &VersionInfo) -> Result<Option<semver::Ver
     }
 }
 
+fn parse_tendermint_version(version_info: &VersionInfo) -> Result<semver::Version, Error> {
+    let module = version_info
+        .build_deps
+        .iter()
+        .find(|&m| m.path.contains(TENDERMINT_MODULE_NAME))
```

Comment:
`contains` should be practically OK, but for extra robustness it might be possible to use `starts_with`, or even match the whole string exactly?

Reply:
We use `contains` because some networks can have their own fork of tendermint, and they're not building on `tendermint/tendermint` but instead on `XXX/tendermint`. I know this is the case often for SDK, e.g., https://github.com/likecoin/cosmos-sdk or https://github.com/irisnet/cosmos-sdk or https://github.com/osmosis-labs/cosmos-sdk but seems for tendermint also: https://github.com/osmosis-labs/tendermint (unclear if they use it, just an example).

I would also like to be more strict here in what strings we accept, but not sure `starts_with` is the right one.

Reply:
If we want to exclude the forks, or any potential `tendermint/tendermint-foo` false matches, it makes even more sense to match with the exact string, doesn't it? A substring match seems too loose as a way to identify modules.

Reply:
Ah, good point! I wasn't really clear in my comment above.

We actually want to _allow_ forks, since they are typically minor improvements, sometimes they include additional modules or exotic features that the vanilla versions don't have. Considering we want to allow forks, what would be a way for us to be more strict?

Reply:
Ok, then let's leave it as it is for the moment. We'll gain more insight once it's out and people are experimenting with it, then we'll adjust the matching criteria to be more strict (or loose) if needed.